src/pages/workspaces/workspace/Data.js

```diff
@@ -762,6 +765,11 @@ const WorkspaceData = _.flow(
                 wdsSchemaError && h(NoDataPlaceholder, {
                   message: 'WDS is unavailable.'
                 }),
+                !wdsSchemaError && h(NoDataPlaceholder, {
+                  message: _.isEmpty(wdsSchema) ? 'No WDS tables have been uploaded.' : '',
```

Comment:
Do we want to change `WDS` or remove `WDS` here? (and I guess in the `WDS is unavailable`?

Seems our messaging to end users won't include `WDS` terminology in them.